.circleci/config.yml

```diff
@@ -57,6 +57,7 @@ jobs:
             python manage.py test --noinput --settings=tock.settings.test
             bandit -r . -x docker_entrypoint.py
             flake8
+            pipenv check --dev
```

Comment:
Great, I'll add this to the SSP. What does this check do for the codebase? Could you link me to documentation or write up a short summary of the feature here?

Reply:
Ref: https://docs.pipenv.org/advanced/#detection-of-security-vulnerabilities

Quick summary: Pipenv leverages the [Safety](https://github.com/pyupio/safety) package to check dependencies against known vulnerabilities. Vulnerabilities are sourced from [Safety-DB](https://github.com/pyupio/safety-db) which are updated once a month by https://pyup.io/.

I would consider this a complement, not a replacement, to active vulnerability scanning a-la gemnasium/snyk/etc.